assets/training/model_management/src/azureml/model/mgmt/utils/exceptions.py

```diff
@@ -80,6 +84,15 @@ def message_format(self) -> str:
         return ModelImportErrorStrings.GIT_CLONE_ERROR
 
 
+class GITConfigError(ClientError):
```

Comment:
Could you also add where to find this authentication token for HF

Reply:
I am planning to add it in azureml-examples notebook. That would be more helpful ig